### PR TITLE
ADDED CATAPI MEDIA TO MESSAGE BODY

### PR DESCRIPTION
**Pull request by ckoegel**

### CODE REVIEW

File: public/index.php [hunk 1]
```diff
@@ -11,12 +11,15 @@
...
 $BW_PASSWORD = getenv("BW_PASSWORD");
+$BW_NUMBER = getenv("BW_NUMBER");
 $BW_MESSAGING_APPLICATION_ID = getenv("BW_MESSAGING_APPLICATION_ID");
 
 $config = new BandwidthLib\Configuration(
     array(
         "messagingBasicAuthUserName" => $BW_USERNAME,
         "messagingBasicAuthPassword" => $BW_PASSWORD
+        //"environment" => BandwidthLib\Environments::CUSTOM,
+        //"baseUrl" => 'https://79fc5f8dded12bf1c7504601d1866997.m.pipedream.net'
```
Comment (ckoegel) [inline public/index.php]:
```suggestion
        //"baseUrl" => 'https://79f1866997.m.pipedream.net'
```

File: public/index.php [hunk 2]
```diff
@@ -90,42 +135,17 @@
...
-  $contents = fread($file, filesize($filename));
-  $messagingClient->uploadMedia($BW_ACCOUNT_ID, "bandwidth_sample_app.jpg", strlen($contents), $contents);
-  fclose($file);
-
-  // delete local image file
-  unlink($mediaName);
-  return $response->withStatus(200);
-});
-
-$app->run();
```
Comment (hamermike) [inline public/index.php:131]: Did you accidentally nuke `$app->run();`?

File: public/index.php [hunk 3]
```diff
@@ -11,12 +11,15 @@
...
 $BW_PASSWORD = getenv("BW_PASSWORD");
+$BW_NUMBER = getenv("BW_NUMBER");
 $BW_MESSAGING_APPLICATION_ID = getenv("BW_MESSAGING_APPLICATION_ID");
 
 $config = new BandwidthLib\Configuration(
     array(
         "messagingBasicAuthUserName" => $BW_USERNAME,
         "messagingBasicAuthPassword" => $BW_PASSWORD
+        //"environment" => BandwidthLib\Environments::CUSTOM,
+        //"baseUrl" => 'https://79f1866997.m.pipedream.net'
```
Comment (hamermike) [inline public/index.php]: Nuke?

**Review by ajrice6713 — APPROVED** [review]

File: public/index.php (hunk 2 above)
Comment (ckoegel) [reply public/index.php:131]: 😬

File: public/index.php (hunk 3 above)
Comment (ckoegel) [inline public/index.php]:
```suggestion
        //"baseUrl" => 'https://custom-url.com'
        // These two lines are used for setting a custom URL to send the request to
        // Documentation can be found [here](https://dev.bandwidth.com/sdks/php.html)
```

Comment (ckoegel) [reply public/index.php]: updated

File: public/index.php [hunk 4]
```diff
@@ -90,42 +137,19 @@
...
-  $messagingClient->uploadMedia($BW_ACCOUNT_ID, "bandwidth_sample_app.jpg", strlen($contents), $contents);
-  fclose($file);
-
-  // delete local image file
-  unlink($mediaName);
-  return $response->withStatus(200);
-});
-
-$app->run();
+$app->run();
```
Comment (ckoegel) [inline public/index.php]:
```suggestion
$app->run();

```

**Review by ajrice6713 — APPROVED** [review]

**Review by abaldwin-Bandwidth — APPROVED** [review]

**Pull request merged**